pfurl/pfurl.py

```diff
@@ -276,25 +282,25 @@ def man(self, **kwargs):
             to a remote REST-like service: """ + Colors.GREEN + """
 
                  ./pfurl.py [--auth <username:passwd>] [--verb <GET/POST>]   \\
-                            --http <IP>[:<port>]</some/path/>
+                            --url http://address.com/path
```

Comment:
I don't like address.com/path/

How about:

<protocol>://<host>[:<port>]</some/path/>

Reply:
ok updated. fore some reason, it wont let me squash the 2 commits i have together

Reply:
nvm i got it